src/Microsoft.Framework.Runtime/NuGet/Authoring/ManifestVersionUtility.cs

```diff
@@ -55,21 +76,20 @@ private static int GetMaxVersionFromMetadata(ManifestMetadata metadata)
             return DefaultVersion;
         }
 
-        private static int VisitObject(object obj)
+        private static int GetVersionFromObject(object obj)
```

Comment:
These codes are still used to decide the manifest schema. However it can be simplified. I filed following issue to track: https://github.com/aspnet/dnx/issues/2199